packages/xrpl/test/models/CredentialDelete.test.ts

```
@@ -0,0 +1,171 @@
+      Sequence: 1337,
+      Flags: 0,
+    } as any
+  })
+
+  it(`verifies valid credentialDelete`, function () {
+    assert.doesNotThrow(() => validateCredentialDelete(credentialDelete))
+    assert.doesNotThrow(() => validate(credentialDelete))
+  })
+
+  it(`throws w/ missing field Account`, function () {
+    delete credentialDelete.Account
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)